Generation/data/Pythia_LHEinput.cmd

```diff
@@ -7,10 +7,8 @@
 ! Adjusted from Pythia example: main42.cmnd
 
 ! 1) Settings that will be used in a main program.
-Main:numberOfEvents = 1            ! number of events to generate
-Main:timesAllowErrors = 3          ! abort run after this many flawed events
-Random:seed = 1234                ! initialize random generator with a seed
-
+Main:numberOfEvents = 100          ! number of events to generate
+Main:timesAllowErrors = 100        ! abort run after this many flawed events
```

Comment:
hum, 100 is maybe a bit too much no?
In a typical job of 10k events we don't expect more than a few failures.